src/fps/utils/BytesHelper.sol

```diff
@@ -0,0 +1,13 @@
+pragma solidity 0.8.15;
+
+library BytesHelper {
+    /// @notice function to grab the first 32 bytes of returned memory
+    /// @param toSlice the calldata payload
+    function getFirstWord(bytes memory toSlice) public pure returns (uint256 value) {
```

Comment:
I don't think this function is used in MultisigProposal.sol, so can we remove this library and introduce it again in the future if we need it?

Reply:
resolving